app/js/controllers/main.js

```diff
@@ -34,6 +50,10 @@ export default class MainController extends BaseController {
         });
     }
 
+    this[p.speechController].start().then(() => {
+      console.log('Speech controller started');
+    });
```

Comment:
I don't think this is likely to happen as it is, as the only interface we expose is events right now - I think it's OK for now.